### ISSUE TITLE
CCSD with complex custom hamiltonian?

### ISSUE THREAD

**Issue by Physfock** [pyscf/pyscf#2339]
Dear users and developers. Is it possible to use ccsd objeci in pyscf with custom compex hamiltonian?

**Comment by tberkel** [comment]
Yes it should be. Here is an example for a CCSD calculation of a custom (real) Hamiltonian: https://github.com/pyscf/pyscf/blob/master/examples/cc/40-ccsd_custom_hamiltonian.py

Depending on the way in which your Hamiltonian is complex, you'll have to choose an appropriate mean-field and CC method to use.

**Comment by Physfock** [comment]
I mean Hamiltonian, Fock matrix with complecx numbers. Provided examples works only with real Hamiltonian and eris

**Comment by tberkel** [comment]
Does it break spin symmetry (like spin-orbit would) or no?
